### ISSUE TITLE
Problem with opening the Rhino from Revit

### ISSUE THREAD

**Issue by marcinwoch** [mcneel/rhino.inside#181]
Hello,

Today I installed Rhino Inside and I have a problem with Licence Manager initialization.
Here is a window which shows up:

![2019-09-11_08h55_50](https://user-images.githubusercontent.com/6567040/64674978-56da0600-d472-11e9-8517-75390d7ef032.png)

Also, I tried to open Revit as Administration, but nothing changed.

I attach the error message "RhinoDotNetCrash":

```
[ERROR] FATAL UNHANDLED EXCEPTION: System.AccessViolationException: An attempt was made to read or write protected memory. It often indicates that another memory is damaged.
w Options.setSSOServer(Options* , Int32 )
w UIFrameworkServices.WebServiceWrapperImp.{dtor}(WebServiceWrapperImp* )
w UIFrameworkServices.WebServiceWrapperImp.__delDtor(WebServiceWrapperImp* , UInt32 A_0)
w UIFrameworkServices.WebServiceWrapper.!WebServiceWrapper()
w UIFrameworkServices.WebServiceWrapper.Dispose(Boolean A_0)

[END ERROR]
```

**Comment by marcinwoch** [comment]
I downloaded again the newest installers and still the same problem.

![image](https://user-images.githubusercontent.com/6567040/64759294-0678ac00-d537-11e9-96af-e5b82eae5429.png)

![image](https://user-images.githubusercontent.com/6567040/64759547-b4845600-d537-11e9-9f51-d726d376e32a.png)

**Comment by scotttd** [comment]
Can you run Rhino 7 WIP outside of Revit, just by itself.  Does it request a license?  Once it runs outside, can you then run it inside Revit?

**Comment by kike-garbo** [comment]
Hi,

Could you please follow instructions there are [here](https://github.com/mcneel/rhino.inside/blob/master/Autodesk/Revit/doc/troubleshooting-rhino-in-revit.md#advanced-search-for-conflicting-plugins) and tell us what you found, it would be so helpfull.

**Comment by marcinwoch** [comment]
> Can you run Rhino 7 WIP outside of Revit, just by itself. Does it request a license? Once it runs outside, can you then run it inside Revit?

Yes. I can easily open just Rhino 7 WIP. The error occurs only when I am in Revit and when I try to click RhinoInside Plugin

**Comment by marcinwoch** [comment]
Gentelmens...
Plugin starts working !!!. I disable all addins from Revit and leave only RhinoInside. 
Now I have to figure out which one cause the problem. I let you know when I will find it.
Thanks a lot

**Comment by marcinwoch** [comment]
Ok. I found the problem. The plugin called Naviate.addin cause a problem. When I disabled it, Rhino.Inside runs perfectly.

**Comment by scotttd** [comment]
We can contact Naviate.  Do you have know someone there that may be able to help?

**Comment by scotttd** [comment]
Do you know which Naviate plugin you are using specifically?

**Comment by eirannejad** [comment]
@marcinwoch Would you be able to open the `Naviate.addin` file in text editor, follow the path set under `<Assembly>` in your windows explorer, and share a screenshot of all the files shipped with the Naviate plugin? Another great adjustment you can do is to show the `Product Version` in the detail view of windows explorer

![2019-09-26_08-54-33](https://user-images.githubusercontent.com/8197916/65704375-4b263c00-e03b-11e9-859f-12c58041f687.gif)

**Comment by ArchitectTaeyoon** [comment]
I just tested the newest version and it still clashes with Dynamo.

Maybe it will have to be run separately after all..

Starting Dynamo - closing it - then firing up Rhino.Inside gives error message.
Starting Rhino.Inside - then starting Dynamo (works up to this stage, but then) - Revit freezes.

**Comment by marcinwoch** [comment]
> @marcinwoch Would you be able to open the `Naviate.addin` file in text editor, follow the path set under `<Assembly>` in your windows explorer, and share a screenshot of all the files shipped with the Naviate plugin? Another great adjustment you can do is to show the `Product Version` in the detail view of windows explorer
> 
> ![2019-09-26_08-54-33](https://user-images.githubusercontent.com/8197916/65704375-4b263c00-e03b-11e9-859f-12c58041f687.gif)

When I was turned on a row you ticked, there is no information about the version. 
Here is a info from Naviate Start Window:

![image](https://user-images.githubusercontent.com/6567040/66186299-33c20100-e682-11e9-9d69-38e376f36c80.png)
